docs/deploy/deployment-manifest.md

```diff
@@ -51,7 +51,7 @@ An example manifest for the released Industrial-IoT IoT Edge modules included in
             "restartPolicy": "always",
             "settings": {
               "image": "mcr.microsoft.com/iotedge/discovery:latest",
-              "createOptions": "{\"Hostname\":\"discovery\",\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}},\"HostConfig\":{\"NetworkMode\":\"host\",\"CapAdd\":[\"NET_ADMIN\"]}}"
+              "createOptions": "{\"Hostname\":\"discovery\",\"NetworkingConfig\":{\"EndpointsConfig\":{\"host\":{}}},\"HostConfig\":{\"NetworkMode\":\"host\",\"CapAdd\":[\"NET_ADMIN\"],\"CapDrop\":[\"CHOWN\",\"SETUID\"]}"
```

Comment:
Should we also add this to the publisher and twin deployments bellow (in the same file) ?

Reply:
added